### PR TITLE
Whitespace consistency to match library arithmetic / generated prefixes

### PR DESCRIPTION
**Pull request by tmilnthorp**

### CODE REVIEW

File: Common/UnitDefinitions/SpecificFuelConsumption.json [hunk 1]
```diff
@@ -37,8 +37,8 @@
       "Localization": [
         {
           "Culture": "en-US",
-          "Abbreviations": [ "g/(kN�s)" ],
-          "AbbreviationsForPrefixes": { "Kilo": [ "kg/(kN�s)" ] }
+          "Abbreviations": [ "g/(kN�s)" ],
```
Comment (tmilnthorp) [inline Common/UnitDefinitions/SpecificFuelConsumption.json:40]: Hmm that's odd. RegEx gone wild.

File: UnitsNet.NanoFramework/GeneratedCode/Quantities/ForceChangeRate.g.cs [hunk 2]
```diff
@@ -288,20 +288,20 @@ private double GetValueInBaseUnit()
...
-                ForceChangeRateUnit.DecanewtonPerMinute => (_value/60) * 1e1d,
+                ForceChangeRateUnit.DecanewtonPerMinute => (_value / 60) * 1e1d,
                 ForceChangeRateUnit.DecanewtonPerSecond => (_value) * 1e1d,
                 ForceChangeRateUnit.DecinewtonPerSecond => (_value) * 1e-1d,
-                ForceChangeRateUnit.KilonewtonPerMinute => (_value/60) * 1e3d,
+                ForceChangeRateUnit.KilonewtonPerMinute => (_value / 60) * 1e3d,
                 ForceChangeRateUnit.KilonewtonPerSecond => (_value) * 1e3d,
-                ForceChangeRateUnit.KilopoundForcePerMinute => (_value*4.4482216152605095551842641431421/60) * 1e3d,
-                ForceChangeRateUnit.KilopoundForcePerSecond => (_value*4.4482216152605095551842641431421) * 1e3d,
+                ForceChangeRateUnit.KilopoundForcePerMinute => (_value * 4.4482216152605095551842641431421/60) * 1e3d,
```
Comment (tmilnthorp) [inline UnitsNet.NanoFramework/GeneratedCode/Quantities/ForceChangeRate.g.cs:296]: TODO: Need to figure out the regex for multiply/divide numbers spacing

File: Common/UnitDefinitions/SpecificFuelConsumption.json (hunk 1 above)
Comment (tmilnthorp) [reply Common/UnitDefinitions/SpecificFuelConsumption.json]: Fixed

File: UnitsNet.NanoFramework/GeneratedCode/Quantities/ForceChangeRate.g.cs (hunk 2 above)
Comment (tmilnthorp) [reply UnitsNet.NanoFramework/GeneratedCode/Quantities/ForceChangeRate.g.cs]: Done

**Comment by tmilnthorp** [comment]
Ok, all consistent now

**Comment by angularsen** [comment]
[Release UnitsNet/4.120.0 · angularsen/UnitsNet](https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F4.120.0)

**Pull request merged**